pkg/network/protocols/http/statkeeper.go

```diff
@@ -85,6 +85,7 @@ func (h *StatKeeper) Close() {
 func (h *StatKeeper) add(tx Transaction) {
 	rawPath, fullPath := tx.Path(h.buffer)
 	if rawPath == nil {
+		telem.malformed.Inc(h.telemetry.proto)
 		h.telemetry.malformed.Add(1)
```

Comment:
why do we need the duplication of telemetries?

Reply:
same for all other locations

Reply:
As mentioned in the PR description, collect metrics via expvar is being deprecated internally, so anything that was only on expvar needs to be exposed via agent telemetry.